manifests/0000_03_config-operator_01_proxy.crd.yaml

```diff
@@ -442,6 +442,8 @@ spec:
                   description: name is the metadata.name of the referenced config
                     map
                   type: string
+              required:
+              - name
```

Comment:
These are correct, this field was manually specified required https://github.com/openshift/api/blob/master/config/v1/types.go#L20-L22